docs/troubleshooting/faq.rst

```diff
@@ -0,0 +1,16 @@
+Frequently Asked Questions
+==========================
+
+I stopped and started my VM, and now JupyterHub doesn't work!
+----------------------------------------------------------------------
+
+Depending on the VM provider that you're using, stopping your VM might
+cause a new External IP address to be generated. If this happens, then your
+old IP address will no longer work.
+
+Figure out what the new External IP address is, then SSH into it and

```

Comment:
You don't need to do this.

Reply:
hmmm, it's the only way I could get it working in my case :-)

Reply:
Interesting. I wonder if this means the service doesn't start automatically on restart?  Can you file a bug separately for that?